### PR TITLE
feat(websocket): wrap firefox web socket too

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: src/server/firefox.ts [hunk 1]
```diff
@@ -185,3 +189,129 @@ export class Firefox implements BrowserType<FFBrowser> {
...
 }
 
+function wrapTransportWithWebSocket(transport: ConnectionTransport, port: number): string {
+  const server = new ws.Server({ port });
+  const guid = platform.guid();
+  const idMixer = new SequenceNumberMixer<{id: number, socket: ws}>();
+  const pendingBrowserContextCreations = new Set<number>();
+  const pendingBrowserContextDeletions = new Map<number, string>();
+  const browserContextIds = new Map<string, ws>();
+  const sessionIds = new Map<string, ws>();
```
Comment (dgozman) [inline src/server/firefox.ts]: nit: sessionToSocket?

File: src/server/firefox.ts [hunk 2]
```diff
@@ -185,3 +189,129 @@ export class Firefox implements BrowserType<FFBrowser> {
...
+  const sockets = new Set<ws>();
+
+  transport.onmessage = message => {
+    if (typeof message.id === 'number') {
+      // Process command response.
+      const value = idMixer.take(message.id);
```
Comment (dgozman) [inline src/server/firefox.ts]: Could you please do `const seqNum = message.id;` and use that below, for clarity?

**Review by dgozman — APPROVED** [review]
Looks good, but lint is not happy.

File: src/server/firefox.ts (hunk 1 above)
Comment (pavelfeldman) [reply src/server/firefox.ts]: Done

File: src/server/firefox.ts (hunk 2 above)
Comment (pavelfeldman) [reply src/server/firefox.ts]: Done

**Pull request merged**